static/scripts/authentication.ts

```diff
@@ -112,4 +121,16 @@ export class AuthService {
       return null;
     }
   }
-}
\ No newline at end of file
+
+  public async getGitHubUserOrgs(): Promise<string[]> {
+    const octokit = await this.getOctokit();
+    const response = await octokit.request("GET /user/orgs");
```

Comment:
This is failing, it seems that the correct endpoint for this request would be "GET /organizations", based on this piece of docs: https://docs.github.com/pt/rest/orgs/orgs?apiVersion=2022-11-28#list-organizations

Reply:
No that's `list all organizations` I'm calling `listForAuthenticatedUser`

https://docs.github.com/en/rest/orgs/orgs?apiVersion=2022-11-28#list-organizations-for-the-authenticated-user

---

I'll use `octokit.rest.listForAuthneticated` method to clear any confusion

Reply:
It's likely failing due to your permissions.

When I was showing you how to run the kernel and plugins I told you to update your kernel app's permissions at it had very very few, I trust you did this yeah?

Reply:
> 2. Supabase auth needs updated with your GitHub app Client ID and secret.

What do you mean with this, exactly? Is it in a secrets format, or an oauth config perhaps?